R/docvars.R

```diff
@@ -125,32 +128,27 @@ subset_docvars <- function(x, i = NULL) {
 
 # internal function to upgrade docvars to modern format
 upgrade_docvars <- function(x, docnames = NULL) {
-    if (sum(is_system(colnames(x))) == 4) 
+    if (sum(is_system(colnames(x))) == 3) 
```

Comment:
Better to check for an `attr(x, "meta"$system$version` and if this is `NULL` we know it's pre-1.5, or later, we can get a version number. (I think I implemented this already in the `dev-corpus2-fix-1613` branch.) Probably we should write a function that checks which version and returns a value. If we don't find the new corpus structure we can return ">=1.4".